supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -122,13 +155,15 @@
 [discrete]
 [[trusted-certificate-authority]]
 ==== image:images/yes.png[yes] trusted Certificate Authority (noun)
-*Description*: A "trusted Certificate Authority" is a third party that creates SSL certificates (CA certificates) used for authentication. It is not to be confused with self-signed certificates. Note the capitalization of Certificate Authority, commonly abbreviated as CA.
+*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA is different from a self-signed certificate in that a self-signed certificate has its own private key and does not need to request a key from a public or private CA. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
+
+A web server uses its public key to obtain a certificate from a trusted CA. The web server stores this certificate in a keystore. During the TLS handshake process, a client checks the validity of the certificate for authentication purposes.
 
 *Use it*: yes
 
-*Incorrect forms*:
+*Incorrect forms*: Self-signed certificate
 
-*See also*:
+*See also*: xref:tls[tls], xref:trusted-certificate-authority[trusted Certificate Authority], xref:certificate-authorities[certificate authorities]
```

Comment:
* This see also for trusted CAs is linking to itself, can you remove that?
* The see also has a link to "certificate authorities" that doesn't seem to exist. Can you remove that?